src/utils/indexer/connector.tsx

```diff
@@ -14,11 +17,13 @@ export class IndexerQuerier {
       // Be advised that if PENUMBRA_INDEXER_CA_CERT is set, then PENUMBRA_INDEXER_ENDPOINT must
       // *lack* an `sslmode` param! This is documented here:
       // https://node-postgres.com/features/ssl#usage-with-connectionstring
-      ...(process.env.PENUMBRA_INDEXER_CA_CERT != null && {
+      ...(ca && {
         ssl: {
-            rejectUnauthorized: true,
-            ca: process.env.PENUMBRA_INDEXER_CA_CERT,
-          },
+          rejectUnauthorized: true,
+          ca: ca.startsWith('-----BEGIN CERTIFICATE-----')
+            ? ca
+            : fs.readFileSync(process.env.PENUMBRA_INDEXER_CA_CERT, 'utf-8'),
+        },
```

Comment:
This is a bit automagicky for my tastes: I'd prefer discrete, explicitly named env vars for filepath or content of the cert. But I appreciate that you were careful to preserve the pre-existing behavior, so if this works for you, I'm happy to merge it!